### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Full revamp of ihatemoney_ynh

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by Salamandar** [comment]
Maybe we could add an upstream version bump in this PR.

File: manifest.json [hunk 1]
```diff
@@ -3,49 +3,47 @@
...
+        "en": "Simple app to manage your collective expenses",
+        "fr": "Gérez simplement vos dépenses collectives"
     },
     "url": "http://ihatemoney.org/",
-    "version": "4.1.5~ynh2",
-    "license": "free",
+    "version": "4.1.5~ynh3",
+    "license": "Beerware",
+    "upstream": {
+        "license": "MIT",
```
Comment (alexAubin) [inline manifest.json]: Eeeh these two license fields are supposed to be the same

**Review by alexAubin — APPROVED** [review]
Super quick review, LGTM

File: manifest.json (hunk 1 above)
Comment (Salamandar) [reply manifest.json]: Why ? I understood that the first one was the license of this repo but upstream.license is the software license.

Comment (alexAubin) [reply manifest.json]: zblerg nope, both are the upstream license ... the fact that we have an "upstream" dict is still pretty new and we kept the old license key at the root of the manifest mainly for backward compat ... Will be much cleaner in manifest v2..

Comment (Salamandar) [reply manifest.json]: OK !

Comment (Salamandar) [reply manifest.json]: I don't have the permissions on the testing branch :(

**Review by Salamandar — APPROVED** [review]

**Pull request merged**